examples/bpk-component-tooltip/examples.js

```diff
@@ -127,9 +127,12 @@ const PopperModifiersExample = () => (
       ariaLabel="Berlin Brandenburg Airport"
       id="my-tooltip"
       target={<Heading>BER</Heading>}
-      popperModifiers={{
-        flip: { enabled: false },
-      }}
+      popperModifiers={[
```

Comment:
is this a change consumers would have to make?
We might need to also update the README.md of the tooltip component to point to the right documentation link for this prop

Reply:
Ah yes you are right good spot on this! Hopefully it will not be massively used property so won't have a big impact but still an breaking change!